marshall/file_read.js

```diff
@@ -0,0 +1,111 @@
+const fs = require('fs');
+const EventEmitter = require('events').EventEmitter;
+const ee = new EventEmitter();
+
+ee.on('file-read', (data) => {
+  console.log('READ: ', data.toString());
+})
+
+
+// Could this function be an example of nested callbacks?
+
+function makeReadFile(txt){
+  return function(){
+    fs.readFile(txt, (err, data) => {
+      console.log(data);
+      ee.emit('file-read', data);
+    });
+  };
+};
+
+var one = makeReadFile('./one.txt');
+var two = makeReadFile('./two.txt');
+var three = makeReadFile('./three.txt');
+
+one();
+two();
+three();
+
```

Comment:
I wouldn't consider this the 'nested callback' solution that Aaron said you could not use.  But it also doesn't satisfy the requirement of ensuring that the files print out in the correct order.  It looks like you are trying to start each file read in the correct order.  But there is no guarantee which order the files will finish their read and print out.